capstone/src/main/webapp/eventlisting.html

```diff
@@ -6,19 +6,32 @@
     <link href="eventDesign.css" type="text/css" rel="stylesheet">
     <script src="script.js"></script>
   </head>
-  <body>
-    <div class="holder">
-      <div class="top">
-        <h3>Event 1</h3>
-      </div> <! end top -->
-      <p>Information About the Event</p>
-      <div class="bottom"></div> <! end bottom -->
-      <div class="middle">
-        <button type="button" onclick="login();">Login</button>
+  <body class="text-center">
+    <ul class="nav">
+      <div class="logo">
+        <a href="/index.html">VOTING CALENDAR</a>
+      </div>
+      <li><a href="/index.html">Home</a></li>
+      <li><a href="/auth">Login</a></li>
+    </ul>
+    <div class="welcome">
+      <h1 class="cover-heading">Available Events</h1>
+      <hr style="height: 2px; border-width: 0; color: gray; background-color: gray;">
+      <p class="lead">
+        Please Add The Following Event(s) To Your Calendar
+      </p>
+    </div>
+      <div class="holder">
```

Comment:
It seems like this line (and the nested tags) are all indented too far right, so that could be causing your issues

Reply:
Hmm, changed it, and to me it looks correct, but still getting the same lint error.